gym/envs/registration.py

```diff
@@ -102,7 +114,15 @@ def __init__(self):
     def make(self, id):
         logger.info('Making new env: %s', id)
         spec = self.spec(id)
-        return spec.make()
+        env = spec.make()
+
+        # Automatically wrap envs with a TimeLimit wrapper, because the limit is part of the env semantics
+        # Omit vnc envs for now, because autowrapping breaks `env.configure`
+        # We will add TimeLimit to all envs after we remove `configure` from the env API
+        if not env.spec.tags.get('vnc'):
```

Comment:
My biggest question in this PR is here. @joschu, would it work for you to autowrap all non-vnc envs for now, and then autowrap the VNC envs once we deprecate `env.configure`?